scripts/coverage-gtapi-mock-drv.sh

```diff
@@ -0,0 +1,27 @@
+#!/bin/bash -e
+
+mkdir coverage_gtapi_mock_drv
+pushd coverage_gtapi_mock_drv
+
+trap "popd" EXIT
+mkdir coverage_files
+mkdir coverage_report
+
+
+cmake .. -DBUILD_TESTS=ON -DCMAKE_BUILD_TYPE=Coverage
+make mock gtapi
+
+lcov --directory . --zerocounters
+
+make test
+find */**/opae-c.dir -iname "*.gcda" | xargs -i cp {} coverage_files
```

Comment:
I get errors, one for each file in opae-c, of the form:
cp: cannot open 'libopae/CMakeFiles/opae-c.dir/src/properties.c.gcda' for reading: Permission denied